packages/terra-action-footer/src/ActionFooterContainer.scss

```diff
@@ -0,0 +1,21 @@
+$default-padding: 0.357142857142857rem;
+
+:local {
+  .action-footer-container {
+    background-color: var(--terra-action-footer-background-color, transparent);
+    border-top: var(--terra-action-footer-border-top, 0.071428571428571rem solid #cfd0d1);
+    display: flex;
+  }
+
+  .has-children {
+    min-height: 2.857142857142857rem;
```

Comment:
Rather than breaking up this and the `is-empty` class below, could you do something like
```css
  .action-footer-container {
    background-color: var(--terra-action-footer-background-color, transparent);
    border-top: var(--terra-action-footer-border-top, 0.071428571428571rem solid #cfd0d1);
    display: flex;
    height: 0.714285714285714rem;
    
    &.has-children {
      height: auto;
      min-height: 2.857142857142857rem;
      padding-bottom: var(--terra-action-footer-padding-bottom, $default-padding);
      padding-left: var(--terra-action-footer-padding-left, $default-padding);
      padding-right: var(--terra-action-footer-padding-right, $default-padding);
      padding-top: var(--terra-action-footer-padding-top, $default-padding);
    }
  }
```

That way you only have to worry about 1 boolean assignment of a classname and don't open yourself up to being in a situation where in the future, you have to try to decide where any future styles that could be added to `action-footer-container` belong? They are always owned at a top-level by the `action-footer-container` class, and its just a matter of whether or not it `has-children`?

Reply:
Also, would we ever want the height/min-height to be themable?

Reply:
@dylan-sftwr good idea

@ChaseSinclair we would need to ask UX. The callouts didn't specify themeable height/min-height

Reply:
Spoke with UX, we're gonna go ahead and make height/min-height themeable, but default it to 10px/40px respectively.